packages/app-content-pages/src/screens/About/About.js

```diff
@@ -57,6 +61,30 @@ function AboutPage() {
 
   return (
     <>
+      <Script
```

Comment:
Because this Script is ultra specific to the About page, I'm going to leave it here. If Sean decides users need access to the widget on more pages, I'll extract it to a different place.